src/compiler/python_generator.cc

```diff
@@ -570,6 +570,92 @@ bool PrivateGenerator::PrintAddServicerToServer(
   return true;
 }
 
+/* Prints out a service class used as a container for static methods pertaining
+ * to a class. This class has the exact name of service written in the ".proto"
+ * file, with no suffixes. Since this class merely acts as a namespace, it
+ * should never be instantiated.
+ */
+bool PrivateGenerator::PrintServiceClass(
+    const grpc::string& package_qualified_service_name,
+    const grpc_generator::Service* service, grpc_generator::Printer* out) {
```

Comment:
Should we call it service class? It is a stub, but is named as service? Do you think something like `SimpleStubClass` is a viable name?

Reply:
I'm not married to the name here since this class is just an implementation detail. My hope was that in the future, this class will also serve as a place to add things that pertain to the server. It's called "Service" here because it corresponds one-to-one to your Protobuf-level `service` definition. If you feel strongly, I'll change it.

Reply:
I'm fine with the naming if we are going to add servicer definition someday. Should we create a TODO issue for it? Or point it to an existing design doc?

Reply:
TODO added.